EasyCalc/EasyCalc.Tests/FunctionComposer_SmokeTests.cs

```diff
@@ -0,0 +1,165 @@
+using FunctionComposeLibrary;
+using System.Reflection;
+using Xunit.Abstractions;
+
+namespace EasyCalc.Tests
+{
+    public class FunctionComposer_SmokeTests
+    {
+        private FunctionComposer _composer;
+        private ITestOutputHelper _output;
+        public FunctionComposer_SmokeTests(ITestOutputHelper output)
+        {
+            _composer = new();
+            _output = output;
+        }
+
+        [Fact]
+        public void CheckSignatureDivisionToParameters()
```

Comment:
Try to use other names in naming test methods. According to this pattern: MethodName_StateUnderTest_ExpectedBehavior